css/style.css

```diff
@@ -582,6 +582,70 @@ HEAD .round-2 {
   color: #ffffff90;
 }
 
+/* skills section */
+
+.skills-section{
+  height: 375px;
+}
+
+.text-center{
```

Comment:
don't change the font-weight to a bootstrap class instead use `.font-weight-bold` to the tag where you need bold.

Reply:
fixed that

Reply:
Still this exists